docs/pages/versions/unversioned/sdk/bar-code-scanner.mdx

```diff
@@ -19,6 +19,8 @@ import {
 import { AndroidPermissions, IOSPermissions } from '~/components/plugins/permissions';
 import { PlatformTags } from '~/ui/components/Tag';
 
+> **warning** **Deprecated:** This module will no longer be available in SDK 51. Use [expo-camera](camera-next.mdx) instead.
```

Comment:
```suggestion
> **warning** **Deprecated:** This module will no longer be available in SDK 51. Use [expo-camera/next](camera-next) instead.
```

Linking nit.

Also, wdyt about stating explicitly that we recommend "Next" version of the package (at least until the next SDK release)?

Reply:
Yeah, that makes sense. Is the link enough, or do we need something like
```markdown
We recommend using expo-camera/next which has barcode scanning built in instead.
```

Reply:
Great idea. Adding a context or broader information how/why is always appreciated.

Reply:
Updated. Thanks for reviewing @Simek 👍